easybuild/easyconfigs/l/libpng/libpng-1.6.34-foss-2018b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpng'
+version = '1.6.34'
+
+homepage = 'http://www.libpng.org/pub/png/libpng.html'
+
+description = "libpng is the official PNG reference library"
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
@crubb You can just drop this one, and use the already existing `libpng-1.6.34-GCCcore-7.3.0.eb`